yarn.lock

```diff
@@ -1173,6 +1216,18 @@
     cosmjs-types "^0.8.0"
     long "^4.0.0"
 
+"@cosmjs/proto-signing@^0.32.2":
```

Comment:
nit: it seems like we now have `0.30.x` and `0.32.x` versions for many of the `@cosmjs/*` deps

Reply:
Good to raise. It looks to be mostly due to leapwallet pinning versions. We could use a resolution to override but that can be brittle.

I'll wait for @samsiegart 's input before merging. Also WRT test plan

Reply:
I tested the deployed preview and looks like it worked. I don't know much about the version thing, it's good to be aware of though, maybe upgrading leap elements would fix it.